File: 4 - JAVA/1 - Estrutura-condicional-duracao-de-jogo.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
	
	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int horaini, horafim, duracao;
		
		System.out.print("Hora inicial: ");
		horaini = sc.nextInt();
		System.out.print("Hora final: ");
		horafim = sc.nextInt();
				
		if (horaini >= horafim) {
			duracao = 24 - (horaini - horafim);
		}
		else {
			duracao = horafim - horaini;
		}
		
		System.out.println("O JOGO DUROU " + duracao + " HORA(S)");
		
		sc.close();

	}
}

File: 3 - Linguagem C++/1 - Estrutura-condicional-duracao-de-jogo.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
   int horaInicial, horaFinal, duracao;

   cout << "Hora inicial: ";
   cin >> horaInicial;
   cout << "Hora final: ";
   cin >> horaFinal;

   if (horaInicial < horaFinal){
        duracao = horaFinal - horaInicial;
   }
   else if (horaInicial >= horaFinal){
       duracao = 24 - (horaInicial - horaFinal);
   }

    cout << "O jogo durou " << duracao << " hora(s)" << endl;

    return 0;
}
